.vscode/settings.json

```diff
@@ -7,5 +7,5 @@
         "candle-pyo3"
     ],
     "python.testing.unittestEnabled": false,
-    "python.testing.pytestEnabled": true
+    "python.testing.pytestEnabled": true,
```

Comment:
Why has this file been modified? This doesn't seem like valid json anymore or am I missing something?

Reply:
Fixed that, my mistake!